### DISCUSSION TITLE
ISO Download Failure

### DISCUSSION THREAD

**Discussion by pnewman587** [Security-Onion-Solutions/securityonion#6624] [Unsupported Versions] [answered]
Hello, 

I've been trying to download the Security Onion 2.3.90-20211213 ISO for the past week. In Windows, downloading it on Chrome or Firefox results in frequent network failures. I was able to retry the download until it completed on Firefox, but this resulted in a file less than 3GB that had a different hash than the one displayed on the downloads page. I have tried installing the ISO on Ubuntu 18.04, but the download has been continuously failing as well. I was wondering if this was due to the AWS server outages. If not, what would you recommend I do to resolve this issue.

Thank you

**Comment by dougburks** [comment, marked as answer]
I just tested and was able to download the full ISO image from https://download.securityonion.net/file/securityonion/securityonion-2.3.90-20211213.iso with no issues whatsoever.

You can check your network for firewalls, proxy servers, or other devices that may be interfering with your download. You might also try a different network altogether.
